### PR TITLE
Update E2E Tests Part 2

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to increase the speed and readability of the e2e tests.

Part of #2003 

# Discussion
- [x] Split up Staking and Capacity Transaction tests
- [x] Remove the Rococo e2e test workflow (no longer viable)

### CODE REVIEW

File: .github/workflows/rococo.yml
Comment (wilwade) [inline .github/workflows/rococo.yml:1]: No longer needed, nor works as the block speed is too slow

File: e2e/capacity/capacity_rpc.test.ts
Comment (wilwade) [inline e2e/capacity/capacity_rpc.test.ts:1]: Just copied out of `e2e/capacity/transactions.test.ts`

File: e2e/capacity/transactions.test.ts [hunk 1]
```diff
@@ -889,110 +888,4 @@ describe('Capacity Transactions', function () {
       });
     });
   });
-
-  describe('Capacity RPC', function () {
```
Comment (wilwade) [inline e2e/capacity/transactions.test.ts:893]: Moved to `e2e/capacity/capacity_rpc.test.ts`

File: e2e/capacity/staking.test.ts [hunk 2]
```diff
@@ -153,91 +153,91 @@ describe('Capacity Staking Tests', function () {
             `expected frozen=${tokenMinStake}, got ${increasedFrozen}`
           );
         });
+      });
```
Comment (wilwade) [inline e2e/capacity/staking.test.ts:156]: Unnesting

File: e2e/capacity/staking.test.ts [hunk 3]
```diff
@@ -286,45 +286,4 @@ describe('Capacity Staking Tests', function () {
       await assert.rejects(failStakeObj.signAndSend(), { name: 'BalanceTooLowtoStake' });
     });
   });
-
-  describe('unstake()', function () {
```
Comment (wilwade) [inline e2e/capacity/staking.test.ts:290]: Moving to `e2e/capacity/unstaking.test.ts`

File: e2e/capacity/unstaking.test.ts
Comment (wilwade) [inline e2e/capacity/unstaking.test.ts:1]: Moved from `e2e/capacity/staking.test.ts`

File: e2e/scaffolding/funding.ts
Comment (wilwade) [inline e2e/scaffolding/funding.ts:1]: Tests run in parallel, so the new tests need separate funding sources

File: e2e/capacity/capacity_rpc.test.ts [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  });
+
+  it('Returns nothing when requesting capacity cost of a non-capacity transaction', async function () {
+    const tx = ExtrinsicHelper.api.tx.msa.retireMsa();
+    const feeDetails: FeeDetails = await ExtrinsicHelper.apiPromise.rpc.frequencyTxPayment.computeCapacityFeeDetails(
+      tx.toHex(),
+      null
+    );
+    assert.notEqual(feeDetails, undefined, 'should have returned a feeDetails');
+    assert(feeDetails.inclusionFee.isNone, 'should have returned something for the inclusionFee');
```
Comment (shannonwells) [inline e2e/capacity/capacity_rpc.test.ts:114]: Shouldn't this message be
```suggestion
    assert(feeDetails.inclusionFee.isNone, 'should have returned None for the inclusionFee');
```

File: e2e/capacity/capacity_rpc.test.ts [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+  it('Returns nothing when requesting pay with capacity call with a non-capacity transaction', async function () {
+    const insideTx = ExtrinsicHelper.api.tx.msa.retireMsa();
+    const tx = ExtrinsicHelper.api.tx.frequencyTxPayment.payWithCapacity(insideTx);
+    const feeDetails: FeeDetails = await ExtrinsicHelper.apiPromise.rpc.frequencyTxPayment.computeCapacityFeeDetails(
+      tx.toHex(),
+      null
+    );
+    assert.notEqual(feeDetails, undefined, 'should have returned a feeDetails');
+    assert(feeDetails.inclusionFee.isNone, 'should have returned something for the inclusionFee');
```
Comment (shannonwells) [inline e2e/capacity/capacity_rpc.test.ts:125]: same as above

**Review by shannonwells — COMMENTED** [review]
Looks like there are a couple of failure messages that need to be changed.

File: e2e/capacity/capacity_rpc.test.ts (hunk 5 above)
Comment (wilwade) [reply e2e/capacity/capacity_rpc.test.ts:125]:
```suggestion
    assert(feeDetails.inclusionFee.isNone, 'should have returned None for the inclusionFee');
```

**Review by JoeCap08055 — APPROVED** [review]
looks good, mostly moving code blocks around, and a new set of unstaking tests. If the pipeline is green, 🚢 it

**Review by mattheworris — APPROVED** [review]
- Read through changes
- Executed tests (All passed)
🚢 it

**Review by aramikm — APPROVED** [review]
Ship it!

**Pull request merged**